packages/ember-data/tests/integration/record-array-manager-test.js

```diff
@@ -118,12 +118,12 @@ test("Should not filter a stor.all() array when a record property is changed", f
     });
   });
 
-  equal(filterdSummary.called.length, 1);
+  equal(filterdSummary.called.length, 0);
```

Comment:
We want to assert that updateRecordArray is also called only once before and after